src/firefly/js/charts/dataTypes/XYColsCDT.js

```diff
@@ -194,7 +194,9 @@ export function setZoom(chartId, chartDataElementId, selection=undefined) {
             if (decimatedUnzoomed || isUndefined(decimatedUnzoomed)) {
                 noFetch = false;
             }
-            dispatchChartOptionsUpdate({chartId, chartDataElementId, updates: {zoom: xyPlotParams.selection, selection: undefined}, noFetch});
+            dispatchChartOptionsUpdate({chartId, chartDataElementId, updates: {zoom: xyPlotParams.boundaries, selection: undefined}, noFetch});
```

Comment:
Can you explain to me why it messed up setting min and max?  I looked at the algorithm used and did not see it messing up the setting.  Maybe I did not understand the algorithm.